### PR TITLE
Docs: Adding a landing page for community platforms.

### PR DESCRIPTION
**Pull request by zbwright**

### CODE REVIEW

File: os/community-platforms.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+[auro]: booting-on-auro.md
+[brightbox]: booting-on-brightbox.md
+[cloud-ca]: booting-on-cloudca.md
+[exoscale]: booting-on-exoscale.md
+[ikoula]: booting-on-ikoula.md
+[nifty]: booting-on-niftycloud.md
+[packet]: booting-on-packet.md
+[rackspace]: booting-on-rackspace.md
+[rimuhosting]: booting-on-launchtimevps.html
```
Comment (athai) [inline os/community-platforms.md]: ``.html`` --> ``.md``

File: os/community-platforms.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Community supported platforms
+
+CoreOS Container Linux supports a growing community of open source contributors. This page lists the most fully developed integrations for Container Linux on cloud and other platforms.
```
Comment (euank) [inline os/community-platforms.md]: it feels disingenuous to call these necessarily "open source contributors".

Many of them are using random closed source scripts to import our images.

"Fully developed" also is a little too much of a claim for quality I think.

My attempt to convey less "support" without it sounding negative.

The CoreOS Container Linux community has provided support for Container Linux on a number of platforms beyond those [officially supported](..) by CoreOS.

The platforms and providers listed below each provide support and documentation for running Container Linux:

File: os/community-platforms.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+* [RimuHosting LaunchtimeVPS][rimuhosting]
+* [VEXXHOST Cloud][vexxhost]
+* [Vultr VPS][vultr]
+
+## Other providers
+
+* [CloudStack][cloudstack]
+* [Eucalyptus][eucalyptus]
+* [libvirt][libvirt]
+* [QEMU][qemu]
```
Comment (euank) [inline os/community-platforms.md]: QEMU should really be considered supported.
Our docs aren't great, but it's honestly our most tested platform.

cc @crawford, is there history here?

Comment (crawford) [reply os/community-platforms.md]: We should move it to a supported platform since it meets our criteria.

File: os/community-platforms.md (hunk 2 above)
Comment (zbwright) [reply os/community-platforms.md]: "officially supported" would simply link to the Container Linux landing page, and the "Cloud Providers" section.

File: os/community-platforms.md (hunk 3 above)
Comment (zbwright) [reply os/community-platforms.md]: moved to the Cloud Providers section of the Container Linux landing page.https://coreosdev.atlassian.net/browse/DOCS-37

File: os/community-platforms.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+# Community supported platforms
+
+The CoreOS Container Linux community has provided support for Container Linux on a number of platforms beyond those [officially supported][official-support] by CoreOS.
```
Comment (euank) [inline os/community-platforms.md]: I think "The Container Linux community" reads better than "The CoreOS Container Linux community".

Comment (zbwright) [reply os/community-platforms.md]: ok

**Pull request merged**